src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -504,21 +507,30 @@ public void sanitizeForSaving() {
         this.courseId = Sanitizer.sanitizeTitle(courseId);
         this.creatorEmail = Sanitizer.sanitizeEmail(creatorEmail);
     }
+    
+    public boolean isJsonValid(String test) {
+        try {
+            new JSONObject(test);
+        } catch (JSONException ex) {
+            // edited, to include @Arthur's comment
+            // e.g. in case JSONArray is valid as well...
```

Comment:
Could remove the top comment :p (line 515), rephrase the bottom comment (line 516) to be `Checks whether JSONArray is valid` and move it to be just above `new JSONArray(test);`